TOOLS.rst

```diff
@@ -168,16 +180,23 @@ main interface to git is via powershell (or mayby Command Prompt) and
 a virtual environment is being used and is active, then the commit
 hook will just work.
 
+For this reason, we recommend using **Anaconda Powershell Prompt** for
```

Comment:
Ah, that seems to be a half-baked scribble during my frustration with issue #10 . This will be fixed in a future merge. The most up-to-date tools.rst is here: https://github.com/ixdat/ixdat/blob/spectroelectrochemistry/TOOLS.rst